docs/dev-tools/zombienet/tutorials/minimal-example.md

```diff
@@ -0,0 +1,235 @@
+}
+
+main().catch(console.error).finally(() => process.exit());
+```
+
+Either way allows you to interact easily with the network and its nodes.
+
+### Checking Metrics
+
+You can also check the metrics of the nodes by accessing the provided links listed by the output as `Prometheus Link`. [Prometheus](https://prometheus.io/){target=_blank} is a monitoring and alerting toolkit that collects metrics from the nodes. By accessing the provided links, you can see the metrics of the nodes in a web interface. So, for example, the following image shows the Prometheus metrics for Bob’s node from the Zombienet test:
+
+![Prometheus metrics for Bob’s node from the Zombienet test](/polkadot-ecosystem-docs-draft/images/dev-tools/zombienet/tutorials/zombienet-tutorials-1.webp)
+
+### Checking logs
+
+To check the nodes’ logs, you can use the provided command listed by the output as 'Log Cmd'. For instance, to check the logs of the `alice` node, you can open a new terminal and run the following command:
+
+```bash
+tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-SEzfCidQ1za4/alice.log
+```
+
+After running this command, you will see the logs of the `alice` node in real-time, which can be useful for debugging purposes. The logs of the `bob` and `collator01` nodes can be checked similarly.
+
+## Running a Test
+
+To run a test against the spawned network, you can use the [Zombienet DSL](../testing.md) to define the test scenario. For example, you can create a file named `minimal-example-test.zndsl` with the following content:
+
+```toml
+Description: Test the basic functionality of the network (minimal example)
+Network: ./minimal-config.toml
+Creds: config
+
+alice: is up
+alice: parachain 100 is registered within 225 seconds
+alice: parachain 100 block height is at least 10 within 250 seconds
+
+bob: is up
+bob: parachain 100 is registered within 225 seconds
+bob: parachain 100 block height is at least 10 within 250 seconds
+
+# metrics
+alice: reports node_roles is 4
+alice: reports sub_libp2p_is_major_syncing is 0
+
+bob: reports node_roles is 4
+
+collator01: reports node_roles is 4
+```
+
+This test scenario checks to verify the following:
+
+- the nodes are running
+- the parachain with ID 100 is registered within a certain timeframe (255 seconds in this example)
+- the parachain block height is at least a certain number within a timeframe (in this case, 10 within 255 seconds)
+- the nodes report metrics 
+
+However, you can define any test scenario following the [Zombienet DSL](../testing.md) syntax.
+
+To run the test, execute the following command:
+
+```bash
+zombienet -p native test minimal-example-test.zndsl
+```
+
+This command will execute the test scenario defined in the `minimal-example-test.zndsl` file on the network. If successful, the terminal will display the test output, indicating whether the test passed or failed.
```

Comment:
Might be nice to provide terminal output here. Feel free to add this in a later PR

Reply:
Will address this topic in a new PR later on 👍